opentelemetry-sdk/src/metrics/reader.rs

```diff
@@ -102,6 +102,28 @@ impl TemporalitySelector for DefaultTemporalitySelector {
     }
 }
 
+/// Configure delta temporality for all [InstrumentKind]
+///
+/// [Temporality::Delta] will be used for all instrument kinds if this
+/// [TemporalitySelector] is used.
+#[derive(Clone, Default, Debug)]
+pub struct DeltaTemporalitySelector {
```

Comment:
If this is only needed for benchmarks it should probably stay in that file rather than being exposed in the public API, unless this is part of a spec.

Reply:
Moved to benchmark file.